UITests/TransferMethod/AddTransferMethodBankAccountTests.swift

```diff
@@ -144,7 +247,7 @@ class AddTransferMethodTests: BaseTests {
                 "Please modify Routing Number to a valid ACH Routing Number of the branch of your bank.")))
     }
 
-    private func setUpBankAccountScreen() {
+    func setUpBankAccountScreen() {
         mockServer.setupGraphQLStubs()
 
         app = XCUIApplication()
```

Comment:
This is already in the BaseTest class, the app is currently starting twice right now.  Please remove this.

Reply:
Thanks, removed